config/routes.rb

```diff
@@ -0,0 +1,6 @@
+get 'signup', to: 'users#new', as: 'signup'
+get 'login', to: 'sessions#new', as: 'login'
+get 'logout', to: 'sessions#destroy', as: 'logout'
+
+resources :users
+resources :sessions
```

Comment:
FYI this is how routes are configured in Rails. We will get to this in a few weeks!